bundles/EcommerceFrameworkBundle/CoreExtensions/ClassDefinition/IndexFieldSelection.php

```diff
@@ -351,6 +351,9 @@ public function denormalize($value, $params = [])
 
             return new ObjectData\IndexFieldSelection($tenant, $field, $preSelect);
         }
+        elseif ($value instanceof ObjectData\IndexFieldSelection) {
```

Comment:
The `else` is not necessary, as the `if` above always returns.
```suggestion
        if ($value instanceof ObjectData\IndexFieldSelection) {
```